drafts/Stormless3.md

```diff
@@ -0,0 +1,322 @@
+
+# Episode 3
+
+
+Build a serverless app on AWS with [Serverless Framework][sf], AWS Lambda, StepFunctions, DynamoDB, API Gateway, and ready-to-use functions from StackStorm Exchange open-source catalog.
+
+[sf]: https://serverless.com/framework
+
+You are watching the Episode Three. 
```

Comment:
I'd just strike this. They know which episode this is - and you already explain what you plan to cover further down